tests/core/contracts/test_contract_call_interface.py

```diff
@@ -346,3 +352,7 @@ def test_call_undeployed_contract(undeployed_math_contract, call):
     with pytest.raises(BadFunctionCallOutput) as exception_info:
         call(contract=undeployed_math_contract, contract_function='return13')
     assert expected_undeployed_call_error_message in str(exception_info.value)
+
+
+def test_call_fallback_function(fallback_function_contract):
+    fallback_function_contract.fallback.call()
```

Comment:
Is there an assertion we can use here on the return value or that `data` was set to `1`?

Reply:
Sorry for the delay. I can assert the return is an empty list()... Not sure what more I can do since fallback function cannot return any value and call() cannot change the data state...